packages/apt/test.sh

```diff
@@ -163,6 +164,7 @@ fi
 
 echo "::endgroup::"
 
+run_test
```

Comment:
I think we don't have to run tests if we don't upgrade it.
How about running test in the only upgrading condition?

Reply:
I mean moving this line to the line160.

Reply:
Woops. i made a mistake in intending to modify it that way.